src/WebJobs.Extensions.CosmosDB/Config/CosmosDBWebJobsBuilderExtensions.cs

```diff
@@ -30,11 +31,12 @@ public static IWebJobsBuilder AddCosmosDB(this IWebJobsBuilder builder)
                 {
                     IConfigurationSection section = config.GetSection(path);
                     section.Bind(options);
-                });                
+                });
 
+            builder.Services.AddAzureClientsCore();
```

Comment:
Do you know what services this adds that are required for AAD support? I'm wondering if it can be scoped down, or if adding all the services is okay...

Reply:
The other Azure extensions call the same method: 
* https://github.com/Azure/azure-sdk-for-net/blob/4162f6fa2445b2127468b9cfd080f01c9da88eba/sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Config/ServiceBusHostBuilderExtensions.cs#L91-L93
* https://github.com/Azure/azure-sdk-for-net/blob/6c2d70003ba77ef2b82d661cf70c641d1fa62338/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Queues/src/StorageQueuesWebJobsBuilderExtensions.cs
* https://github.com/Azure/azure-sdk-for-net/blob/f52c5e7fd7ca0f631a3e069dafb7ff0dd47affce/sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubWebJobsBuilderExtensions.cs#L85